x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/RoleDescriptorTests.java

```diff
@@ -773,6 +773,7 @@ public void testParseRemoteIndicesPrivilegesFailsWhenUntrustedRemoteClusterEnabl
                 "test",
                 XContentHelper.createParser(XContentParserConfiguration.EMPTY, new BytesArray(json), XContentType.JSON),
                 false,
+                false,
```

Comment:
Since we are adding a new parameter to `RoleDescriptor.parse`, I think we should add an unit test for the method in this file to directly test its effect.

Reply:
Will add one.